src/Collections/Instruments.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Pantheon\Terminus\Collections;
+
+class Instruments extends TerminusCollection
+{
+    /**
+     * @var User
+     */
+    public $user;
+    /**
+     * @var string
+     */
+    protected $collected_class = 'Terminus\Models\Instrument';
+
+    /**
+     * Object constructor
+     *
+     * @param array $options Options to set as $this->key
+     */
+    public function __construct($options = [])
+    {
+        parent::__construct($options);
+        $this->user = $options['user'];
```

Comment:
@TeslaDethray, @greg-1-anderson: I'm pondering using the DI container to manage the handing off the user object. There's a lot of boilerplate like this that could be eliminated. That would mean fewer tests too. 

The Authorizer would probably be in charge of adding the authenticated user object to the container. Any user-dependent model or collection that was instantiated before that happened would probably throw an exception somewhere. Thoughts?